packages/node/src/integrations/tracing/fastify/fastify-otel/index.js

```diff
@@ -33,6 +33,7 @@ SOFTWARE.
 /* eslint-disable jsdoc/require-jsdoc */
 /* eslint-disable max-lines */
 /* eslint-disable no-param-reassign */
+/* eslint-disable import/named */
```

Comment:
l: why do we need this new disable directive?

Reply:
> [import/export](https://www.npmjs.com/package/eslint-plugin-import) got a false positive, which affects most of our index barrel files (I didn't want to disable it entirely, so I added a global disable in each affected barrel file)
tracked here: https://github.com/import-js/eslint-plugin-import/issues/703